### PR TITLE
Added missing camera methods to Snapshotter

### PR DESCRIPTION
**Pull request by julianrex**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-ios/blob/main/CONTRIBUTING.md#contributor-license-agreement).

## Pull request checklist:
 - [X] Briefly describe the changes in this PR.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [X] Document any changes to public APIs.
 - [X] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] ~Add an entry inside this element for inclusion in the `mapbox-maps-ios` changelog: `<changelog></changelog>`.~
 - [ ] Update the migration guide, API Docs, Markdown files - Readme, Developing, etc

### Summary of changes

This PR adds 2 camera methods to `Snapshotter` to align with Android:
```
func coordinateBounds(for camera: CameraOptions) -> CoordinateBounds

func camera(for coordinates: [CLLocationCoordinate2D],
                       padding: UIEdgeInsets,
                       bearing: Double?,
                       pitch: Double?) -> CameraOptions
```

/cc @kiryldz

### CODE REVIEW

File: Sources/MapboxMaps/Snapshot/Snapshotter.swift [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import UIKit
+import CoreLocation
 
 #if canImport(MapboxMapsFoundation)
```
Comment (neelmistry94) [inline Sources/MapboxMaps/Snapshot/Snapshotter.swift:4]: We should be able to remove these `canImport` statements now right?

File: Sources/MapboxMaps/Snapshot/Snapshotter.swift [hunk 2]
```diff
@@ -202,6 +203,37 @@ public class Snapshotter {
...
+
+    // MARK: - Camera
+
+    /// Returns the coordinate bounds corresponding to a given `CameraOptions`
+    ///
+    /// - Parameter camera: The camera for which the coordinate bounds will be returned.
+    /// - Returns: `CoordinateBounds` for the given `CameraOptions`
+    public func coordinateBounds(for camera: CameraOptions) -> CoordinateBounds {
+        return mapSnapshotter.coordinateBoundsForCamera(
+            forCamera: MapboxCoreMaps.CameraOptions(camera))
```
Comment (neelmistry94) [inline Sources/MapboxMaps/Snapshot/Snapshotter.swift:215]: this spacing is hard and weird to read. Can't this just be on one line

File: Sources/MapboxMaps/Snapshot/Snapshotter.swift [hunk 3]
```diff
@@ -202,6 +203,37 @@ public class Snapshotter {
...
+    ///   - padding: The new padding to be used by the camera.
+    ///   - bearing: The new bearing to be used by the camera.
+    ///   - pitch: The new pitch to be used by the camera.
+    /// - Returns: A `CameraOptions` that fits the provided constraints
+    public func camera(for coordinates: [CLLocationCoordinate2D],
+                       padding: UIEdgeInsets,
+                       bearing: Double?,
+                       pitch: Double?) -> CameraOptions {
+        return CameraOptions(
+            mapSnapshotter.cameraForCoordinates(
```
Comment (neelmistry94) [inline Sources/MapboxMaps/Snapshot/Snapshotter.swift:231]: same as above, this formatting looks weird. this should go up a line and everything else should be indented accordingly

**Review by neelmistry94 — APPROVED** [review]
mainly nits

File: Sources/MapboxMaps/Snapshot/Snapshotter.swift (hunk 1 above)
Comment (julianrex) [reply Sources/MapboxMaps/Snapshot/Snapshotter.swift:4]: Yes, let's tackle as followup

**Pull request merged**